events-sample/AndroidManifest.xml

```diff
@@ -0,0 +1,31 @@
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+    package="org.gearvrf.input.events"
+    android:versionCode="1"
+    android:versionName="1.0" >
+
+    <uses-permission android:name="android.permission.INTERNET"/>
```

Comment:
Ah, I based this out of the RenderableView project. In any case I've uploaded a new pull request to fix both projects:

https://github.com/gearvrf/GearVRf-Demos/pull/40